include/motor-control/core/tasks/gear_tmc2160_motor_driver_task.hpp

```diff
@@ -87,6 +87,15 @@ class MotorDriverMessageHandler {
         }
     }
 
+    void handle(const can::messages::ReadMotorDriverErrorStatusRequest& m) {
+        LOG("Received read motor driver error register request");
+        uint32_t data = 0;
+        std::array tags{spi::utils::ResponseTag::IS_ERROR_RESPONSE};
```

Comment:
doesn't look like the `IS_ERROR_RESPONSE` tag is handled in the `TransactResponse` message handler above

Reply:
Great catch!